src/main/java/hudson/tasks/junit/TestAction.java

```diff
@@ -33,6 +33,8 @@
  * <li>index.jelly: included at the top of the test page</li>
  * <li>summary.jelly: included in a collapsed panel on the test parent page</li>
  * <li>badge.jelly: shown after the test link on the test parent page</li>
+ * <li>classresult-tableheader.jelly: allows additional table headers to be shown in the test class' table of test methods</li>
+ * <li>classresult-tablerow.jelly: allows additional table cells to be shown in the test class' table of test methods</li>
```

Comment:
```suggestion
 * <li>classresult-tableheader.jelly: allows additional table headers to be shown in the test class's table of test methods</li>
 * <li>classresult-tablerow.jelly: allows additional table cells to be shown in the test class's table of test methods</li>
```

Reply:
Thank you for the grammar fix, will include in next commit!